internal/client.go

```diff
@@ -94,6 +94,17 @@ type (
 		//	- InternalServiceError
 		SignalWorkflow(ctx context.Context, workflowID string, runID string, signalName string, arg interface{}) error
 
+		// SignalWithStartWorkflow sends a signals to a workflow in execution, if workflow is not running or not found,
+		// it starts workflow then sends the signal in transaction.
```

Comment:
suggested update:
SignalWithStartWorkflow sends a signal to a running workflow. If the workflow is not running or not found, it starts the workflow and then sends the signal in transaction.